### PR TITLE
Release 0.19.0

### PR DESCRIPTION
**Pull request by lkysow**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-## Unreleased
+## Upcoming
```
Comment (ishustava) [inline CHANGELOG.md:1]: Is this change intentional?

Comment (lkysow) [reply CHANGELOG.md:1]: nope!

**Comment by lkysow** [comment]
> The [`Chart.yaml`](https://github.com/hashicorp/consul-helm/blob/master/Chart.yaml#L3) will need updating to the new version as well.

yep, it's here https://github.com/hashicorp/consul-helm/pull/415/files#diff-6eed8b49cc35c68013f7f840c2e9dbc2R3 or is there another piece?

**Comment by adilyse** [comment]
Somehow I just missed it!
